extensions/HyperVExtension/src/HyperVExtension/Models/VirtualMachineCreation/DotNetZipArchiveProvider.cs

```diff
@@ -12,6 +12,10 @@ namespace HyperVExtension.Models.VirtualMachineCreation;
 /// <see cref="ZipArchive"/> is used by Hyper-V Manager's VM Quick Create feature. This gives us parity, but in the future it is expected that faster/more efficient
 /// archive extraction libraries will be added.
 /// </summary>
+/// <remarks>
+/// .Net cores ZipFile and ZipArchive implementations for extracting large files (Gbs) are slow when used in Dev Homes Debug configuration.
```

Comment:
Do we want gigabits (Gbs) or gigabytes (GBs)?
```suggestion
/// .Net core's ZipFile and ZipArchive implementations for extracting large files (Gbs) are slow when used in Dev Homes Debug configuration.
```